src/datepicker/adapters/ngb-date-native-adapter.spec.ts

```diff
@@ -1,4 +1,5 @@
 import {NgbDateNativeAdapter} from './ngb-date-native-adapter';
+import {type} from 'os';
```

Comment:
What is this import? I guess it was added by accident?

Reply:
Yep, thanks